src/System.Diagnostics.Debug/tests/DebugTests.cs

```diff
@@ -22,14 +25,14 @@ public void Asserts()
             VerifyAssert(() => { Debug.Assert(false, "assert passed", "nothing is wrong {0} {1}", 'a', 'b'); }, "assert passed", "nothing is wrong a b");      
         }
 
-        [Fact]
+        [ConditionalFact(nameof(IsNotDesktopJob))]
```

Comment:
The implementation assembly on desktop and corefx is different, and the tests here are testing the corefx behavior. These tests are not valid on desktop.. Will add a comment.